api/helpers/imageUtils.js

```diff
@@ -1,12 +1,17 @@
 const fileSystem = require("fs");
 
-exports.imageDelete = async fileName => {
-  await fileSystem.unlinkSync("./productsImages/" + fileName);
+exports.imageDelete = fileName => {
+  fileSystem.unlink("./productsImages/" + fileName, error => {
+    if (error) throw error.message;
```

Comment:
it should throw new error, ideally you'd create new Error type and use it here, something like "FileOperationError"